### PR TITLE
Bg fix project dropdown feedback

### PR DESCRIPTION
**Pull request by neelxie**
#### What does this PR do?

Work on dropdown feedback

#### Description of Task to be completed?
- Work on dropdown padding and colors for projects form

#### How should this be manually tested?

* **Before you review this PR take time to read these below**
> https://web.archive.org/web/20170528070441/http://msdn.microsoft.com/en-us/library/ms535877(v=vs.85).aspx
> https://stackoverflow.com/questions/7208786/how-to-style-the-option-of-an-html-select-element

#### screenshots
image

### CODE REVIEW

**Comment by Mubangizi** [comment]
Noo dont give it a blue color, it looks Ugly

**Comment by neelxie** [comment]
> Noo dont give it a blue color, it looks Ugly

@Mubangizi your feedback is not actionable, If you dont want a blue then suggest a color instead of denouncing one and not proposing an alternative.

**Comment by StephenAraka** [comment]
For consistency, you could use similar styling as we did for this 👇
Screenshot from 2020-04-08 15-28-50

**Comment by neelxie** [comment]
> For UX consistency, you could use similar styling as we did for this 👇 ... (with similar hover effect)
> Screenshot from 2020-04-08 15-28-50

@StephenAraka thank you for highlighting this.

**Comment by MutegekiHenry** [comment]
> For UX consistency, you could use similar styling as we did for this point_down ... (with similar hover effect)
> Screenshot from 2020-04-08 15-28-50

yeah sounds more efficient in this case

**Comment by neelxie** [comment]
> For UX consistency, you could use similar styling as we did for this 👇 ... (with similar hover effect)
> Screenshot from 2020-04-08 15-28-50

@StephenAraka @MutegekiHenry  this suggestion doesnt work, consider that styling can not work for select elements.

**Comment by neelxie** [comment]
> > For UX consistency, you could use similar styling as we did for this 👇 ... (with similar hover effect)
> > Screenshot from 2020-04-08 15-28-50
> 
> @StephenAraka @MutegekiHenry this suggestion doesnt work, consider that styling can not work for select elements.

Kindly do go through the PR description for more clarity.

File: src/components/UserProjectsPage/UserProjectsPage.css [hunk 1]
```diff
@@ -47,3 +47,10 @@ section option{
     font-size: 16px;
     color: black;
 }
+
+option{
```
[the remark this answers is not in the document]
Comment (neelxie) [reply src/components/UserProjectsPage/UserProjectsPage.css]: @Mubangizi thank you I have that fixed

**Comment by Mubangizi** [comment]
@neelxie dont make the dropdown name `bold` please.
Screenshot from 2020-04-14 13-56-58
And i dont think this PR is yet ready for review. i dont.

**Comment by neelxie** [comment]
> The limited styling for option tags is an interesting one... We can let that go for now
> Issues you should fix however...
> 
> 1. You have a conflicting classname...
>    Screenshot from 2020-04-14 14-21-05
>    This is being used already in the `SideNav`. Consider having class names that are as descriptive as possible to avoid clashes... e.g. `ClusterNameOption` for the above
> 2. Solve Linting errors. (to decrease our debt when it comes to optimization)
> 3. Conflicts

@StephenAraka this has been fixed

**Comment by neelxie** [comment]
> > > For UX consistency, you could use similar styling as we did for this point_down ... (with similar hover effect)
> > > Screenshot from 2020-04-08 15-28-50
> > 
> > 
> > @StephenAraka @MutegekiHenry this suggestion doesnt work, consider that styling can not work for select elements.
> 
> Okay, so according to what you are saying and what i have found out, it seems `background-color` might not be able to work. then you could try an alternative to `select` option instead, some thing like using `Lists`. Like in this [example](http://jsfiddle.net/8FydL/445/) maybe

@Mubangizi did you notice that this very example makes use of a third party framework? MooTools.

File: src/components/UserProjectsPage/index.jsx [hunk 2]
```diff
@@ -128,9 +127,9 @@ class UserProjectsPage extends React.Component {
     } = this.props;
     const userId = data.id;
     const clustersList = clusters.length > 0
-      && clusters.map((item, i) => (
-        <option key={i} value={item.id}>{item.name}</option>
-      ));
+        && clusters.map((item, i) => (
+          <option className="ClusterNameOption" key={i} value={item.id}>{item.name}</option>
```
Comment (mrwagaba) [inline src/components/UserProjectsPage/index.jsx:131]: Not advisable to use the item index as the key. Try using `short-id`.

**Pull request merged**